operator/api/v1alpha1/manifest_types.go

```diff
@@ -56,7 +56,7 @@ type ManifestSpec struct {
 
 	// CRDs specifies the custom resource definitions' ImageSpec
 	// +kubebuilder:validation:Optional
-	CRDs types.ImageSpec `json:"crds"`
+	CRDs *types.ImageSpec `json:"crds"`
```

Comment:
I think instead of a pointer, we should default to a layer that does nothing instead. That way you do not have to use a pointer.